notify/grafana_alertmanager.go

```diff
@@ -140,18 +152,19 @@ type Configuration interface {
 	DispatcherLimits() DispatcherLimits
 	InhibitRules() []InhibitRule
 	MuteTimeIntervals() []MuteTimeInterval
-	ReceiverIntegrations() (map[string][]*Integration, error)
-	BuildReceiverIntegrationsFunc() func(next *GrafanaIntegrationConfig, tmpl *templates.Template) (Notifier, error)
+	Receivers() map[string]*APIReceiver
```

Comment:
I think this can be a slice because information about receiver name is in APIReceiver.

Reply:
By doing this you will be able to remove this confusing method

https://github.com/grafana/grafana/pull/66901/files#diff-4aa6a5bc542bc278b3cddc81fe825af177f1f354c9dbb08d0f01d86b7dbe414eR313-R319